tests/aio-create.sh

```diff
@@ -214,15 +220,26 @@ pushd /opt/openstack-ansible
   # Disable tempest on older releases
   sed -i '/.*run-tempest.sh.*/d' scripts/gate-check-commit.sh  # Disable the tempest run
 
-  # NOTE(tonytan4ever): pin pip to the last working version before pip 10
+  # NOTE(tonytan4ever): pin pip to the last working version before pip 10	
   pip install -U -r /opt/openstack-ansible/global-requirement-pins.txt
 
   # Pin python-ldap so it stops breaking everything
+  # Fixing missing netbase packages
   echo "python-ldap<3;python_version=='2.7'" >> /opt/openstack-ansible/global-requirement-pins.txt
+  sudo apt-get -o Dpkg::Options::="--force-confmiss" install --reinstall netbase
```

Comment:
Can you clarify why this is needed?

Reply:
This is for this error from _Add in cinder devices types_  in _os_cinder_ role:

```
File \"/openstack/venvs/cinder-16.0.14/lib/python2.7/site-packages/eventlet/support/dns/rdtypes/IN/WKS.py\", line 23, in <module>", "    _proto_tcp = socket.getprotobyname('tcp')", "socket.error: protocol not found"], "stdout": "", "stdout_lines": []
```

This is also per @andymcc's advice: https://rackspace.slack.com/archives/C8TEU9GUT/p1526996313000235

Reply:
@tonytan4ever OK, understood. Now can you not just install it as per normal (as it doesn't appear to be in the nodepool images by default) without doing all this install --reinstall business? We may want to check w/ @odyssey4me if that package should be installed by default, as it does seem sort of critical.